code/modules/vending/clothingbooth_items.dm

```diff
@@ -84,21 +126,106 @@
 	category = "Costumes"
 	cost = 100
 
+/datum/clothingbooth_item/russsianmob
+	name = "Russian Mobster Suit"
+	path = /obj/item/clothing/under/misc/rusmob
+	category = "Costumes"
+	cost = 125
+
+/datum/clothingbooth_item/columbianmob
+	name = "Columbian Mobster Suit"
+	path = /obj/item/clothing/under/misc/colmob
+	category = "Costumes"
+	cost = 125
+
+/datum/clothingbooth_item/mobilesuit
+	name = "Mobile Robot Suit"
+	path = /obj/item/clothing/under/gimmick/mobile_suit
+	category = "Costumes"
+	cost = 10000
+
+/datum/clothingbooth_item/mobilesuithelmet
+	name = "Mobile Robot Helmet"
+	path = /obj/item/clothing/head/mobile_suit
+	category = "Costumes"
+	cost = 1000
+
 //Formal
-/datum/clothingbooth_item/dress
-	name = "Little Black Dress"
-	path = /obj/item/clothing/under/misc/dress
+
+/datum/clothingbooth_item/shirtandpantsblack
+	name = "Shirt And Black Pants"
```

Comment:
Looks like convention is to not capitalize "and" (i.e. "Shirt and Black Plants"). Looking at e.g "Skull Mask and Cloak", "Shirt and Red Tie".

Reply:
done